packages/indicators/src/builders/buildArithmetic.ts

```diff
@@ -45,8 +70,20 @@ const fetchAnalyticClusters = async (
   const checkClusterIncludesAllElements = (cluster: AnalyticCluster) =>
     allElements.every(member => member in cluster.dataValues);
 
+  const replaceAnalyticValuesWithDefaults = (cluster: AnalyticCluster) => {
+    const returnDataValues = { ...cluster.dataValues };
+    Object.keys(defaultValues).forEach(code => {
+      returnDataValues[code] = returnDataValues[code] ?? defaultValues[code];
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Nice :+1:

Reply:
This is a great syntax!